### PR TITLE
Guidelines modal

### PR DESCRIPTION
**Pull request by 0tuedon**
How to test for first timers
1. Make sure you don't have any reviews either pending, inactive or active, if you have time you could negate the code in queuetable.tsx line 194 `  if (multipleStatusData.length > 0) {`

How to test for old timers
1. There is an orange review Guidlines button at the top of the editor, click that and you have your modal

we could change the design going forward.

TESTED.

### CODE REVIEW

File: src/components/editTranscript/EditTranscript.tsx [hunk 1]
```diff
@@ -91,18 +94,33 @@ const EditTranscript = ({
...
+          <Box>
+            <Button
+              colorScheme="orange"
+              onClick={openGuidelines}
+              size="xs"
+              ml="auto"
+              display="block"
+              variant="solid"
+            >
+              Review Guidelines
```
Comment (kouloumos) [inline src/components/editTranscript/EditTranscript.tsx:107]: I think that an icon like `FaBook` could be a good match here at the beginning of the text

File: src/components/alerts/ReviewGuidelinesAlert.tsx [hunk 2]
```diff
@@ -0,0 +1,231 @@
...
+                <Text>
+                  Maintain completeness while ensuring readability by
+                  eliminating mid-sentence rephrasings and non-essential lines.
+                </Text>
+              </ListItem>
+            </UnorderedList>
+          </Flex>
+
+          <Flex my={2} px={5} flexDir={"column"} gap={2}>
+            <Text>Transcript Structure:</Text>
```
Comment (kouloumos) [inline src/components/alerts/ReviewGuidelinesAlert.tsx:96]: Some of those headings have ":" and others do not. I think it's better without, so just "Transcript Structure".

**Review by kouloumos — COMMENTED** [review]
Modal looks good! except in my small 13 inches laptop screen, where it has some weird behavior. Could we do something to take into consideration the max height of the screen and adjust?
Screencastfrom2024-01-3013-57-07-ezgif com-video-to-gif-converter

Also, I left some comments inline regarding implementation details.

File: src/components/tables/QueueTable.tsx [hunk 3]
```diff
@@ -186,7 +191,11 @@ const QueueTable = () => {
                   await retryLoginAndClaim(transcriptId);
                   return;
                 }
-                router.push(`/reviews/${data.id}`);
+                if (multipleStatusData.length > 0) {
+                  router.push(`/reviews/${data.id}`);
+                } else {
+                  router.push(`/reviews/${data.id}?first-review=true`);
```
[the remark this answers is not in the document]
Comment (0tuedon) [reply src/components/tables/QueueTable.tsx]: yh do we want them to see the modal even when they have claimed the first time?

[the remark this answers is not in the document]
Comment (kouloumos) [reply src/components/tables/QueueTable.tsx]: What I am saying is that the current implementation doesn't make sure that they actually clicked the "I Agree" button.

On second thought, I am just making things more complicated without a reason. Let's keep it as it is for now - it's a very good start and much better than what we have currently.

**Comment by kouloumos** [comment]
@0tuedon modal looks good! I pushed some UI changes alongside a CTA for our discord server, please check them out and once you approve them we can merge this.

Also with ef0196a463f4b66dd4111fe941ef226102eee59f , I renamed the component to a more appropriate name as you moved it to the `/modals` directory.

**Comment by 0tuedon** [comment]
> @0tuedon modal looks good! I pushed some UI changes alongside a CTA for our discord server, please check them out and once you approve them we can merge this.
> 
> Also with [ef0196a](https://github.com/bitcointranscripts/transcription-review-front-end/commit/ef0196a463f4b66dd4111fe941ef226102eee59f) , I renamed the component to a more appropriate name as you moved it to the `/modals` directory.


Screenshot 2024-02-01 at 14 19 16
Looks good My only concern is, why on the right?

**Comment by kouloumos** [comment]
> My only concern is, why on the right?

Just a design decision. I believe that aligning the call to action to the right helps create a visual separation and draws attention to it. It signals that it's a distinct and important piece of information.

**Comment by kouloumos** [comment]
@0tuedon I think that based on what we discussed about atomic commits, we should squash the last 5 commits into 1df9f2d7960b1b663aa5d3b9e762e74611c174be before merging.

**Comment by 0tuedon** [comment]
> @0tuedon I think that based on what we discussed about atomic commits, we should squash the last 5 commits into [32243a3](https://github.com/bitcointranscripts/transcription-review-front-end/commit/32243a38c9a670d8a64d8ea631cf823a0b82a01f) before merging.

yh we should

**Pull request merged**